qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -700,6 +700,31 @@ public void allowsUppercaseAbbreviationExceptions() throws Exception {
     }
 
     /**
+     * CheckstyleValidator can reject non diamond operator usage.
+     * @throws Exception If error
+     */
+    @Test
+    public void rejectsNonDiamondOperatorUsage() throws Exception {
+        final String file = "InvalidDiamondsUsage.java";
+        final Collection<Violation> results = this.runValidation(
+            file, false
+        );
+        final String violation = StringUtils.join(
+            "[27]: Better to use diamond operator where possible ",
```

Comment:
@vkuchyn better message is "Use diamond operator"
This will be printed to the developer :)